src/Umbraco.Core/Services/Implement/ContentService.cs

```diff
@@ -606,6 +606,29 @@ public IEnumerable<IContent> GetPublishedChildren(int id)
                 return GetPagedLocked(null, pageIndex, pageSize, out totalChildren, filter, ordering);
             }
         }
+        public IEnumerable<IContent> GetPagedDescendants(int id, long pageIndex, int pageSize, 
```

Comment:
This code can be shared in one method which can accept a bool as to whether or not to include the count